### ISSUE TITLE
NTR: Rabies virus RT-PCR positive (Lab: bacteriology, virology, parasitology, fungal)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8074]
New term request Rabies virus RT-PCR positive (samant_110131003557): Reverse transcriptase polymerase chain reaction; RNA  UMLS: 3687179

**Comment by pnrobinson** [comment]
@MickeySegal
what is the tissue in which the test is being performed?

**Comment by MickeySegal** [comment]
https://www.cdc.gov/rabies/php/laboratories/diagnostic.html
"The LN34 PCR test for rabies is a newer diagnostic test that uses real-time reverse transcriptase polymerase chain reaction (real-time RT-PCR) methodology to detect the presence of rabies virus genetic material.

The best tissue for rabies testing by any test is the brain, and rabies rule-out by the LN34 test requires a full cross-section of the brainstem and representative samples from the cerebellum. Skin biopsy from the nape of the neck and saliva samples can be tested by LN34 to detect rabies in suspect antemortem human rabies cases. This innovative test targets a highly conserved region of the virus genome including the leader region and nucleoprotein gene, which ensures robust detection across all viruses that can cause rabies."

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000961
- label: Positive brain-tissue rabies nucleic acid test
- parents: [Positive infectious agent test (HP:0430030)](https://hpo.jax.org/app/browse/term/HP:0430030)
- pmid: 
- def: Detection of rabies nucleic acid in the sputum or pharyngeal sample by a method such as polymerase chain reaction.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8074
- synonyms:

**Comment by MickeySegal** [comment]
The use of brain tissue seems to be only postmortem.  I'd prefer this to be saliva as described in the quote above.

**Comment by pnrobinson** [comment]
@MickeySegal 

Adding new term: 
- ID: HP:6001111
- label: Positive saliva rabies nucleic acid test
- parents: [Positive oropharangeal infectious agent test (HP:6000579)](https://hpo.jax.org/app/browse/term/HP:6000579)
- pmid: 
- def: Detection of rabies nucleic acid in saliva by a method such as polymerase chain reaction.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8074
- synonyms:
